internal/pkg/manifest/lb_web_svc.go

```diff
@@ -22,13 +22,19 @@ const (
 const (
 	// LogRetentionInDays is the default log retention time in days.
 	LogRetentionInDays     = 30
-	defaultHealthCheckPath = "/"
+	DefaultHealthCheckPath = "/"
 )
 
 var (
 	errUnmarshalHealthCheckArgs = errors.New("can't unmarshal healthcheck field into string or compose-style map")
 )
 
+// Durationp is a utility function used to convert a time.Duration to a pointer. Useful for YAML unmarshaling
+// and template execution.
+func Durationp(v time.Duration) *time.Duration {
```

Comment:
It's slightly odd that we have a `manifest.Durationp` function exposed, can we instead replicate this function in both the `stack` and `manifest` pkgs while keeping it private?